src/Whoops/Resources/js/whoops.base.js

```diff
@@ -50,13 +50,25 @@ Zepto(function($) {
 	  var clipEl = document.getElementById("copy-button");
 	  var clip = new ZeroClipboard( clipEl );
 	  var $clipEl = $(clipEl);
+	  var $clipHelpEl = $("#help-clipboard");
 
 	  // show the button, when swf could be loaded successfully from CDN
 	  clip.on("load", function() {
 		  $clipEl.show();
+		  $clipHelpEl.show();
 	  });
   }
-  
+
+  $(document).on('click', '.Whoops-hider', function () {
+	  var $container = $('.Whoops.container');
+	  if ($container.hasClass('hidden') === false) {
+		  $container.hide().addClass('hidden');
```

Comment:
I added the .container.hidden rule, per your input.

I don't agree with you about the hidden attribute, "This attribute must not be used to hide content that could legitimately be shown." :)